src/sentry/api/validators/sentry_apps/schema.py

```diff
@@ -43,6 +43,8 @@
                 "type": {"type": "string", "enum": ["select"]},
                 "label": {"type": "string"},
                 "name": {"type": "string"},
+                "async": {"type": "boolean"},
```

Comment:
do you think we could add these options in the `test_issue_link` test?

Reply:
@MeredithAnya Which test is this? can't find it.

Reply:
`validators/sentry_apps/test_issue_link.py` p sure

Reply:
@MeredithAnya My bad, I didn't realize it was a file. Yes, I will update it.